### DISCUSSION TITLE
Combine group bar chart and line chart

### DISCUSSION THREAD

**Discussion by nguyencaochien** [grafana/grafana#54739] [Dashboards, Panels & Visualizations]
Can i create chart like this image? 
<img width="933" alt="Screen Shot 2022-09-06 at 15 22 50" src="https://user-images.githubusercontent.com/24190909/188585511-a24cb18c-9e29-4697-824b-a287f19eb037.png">

Visualization: bar chart

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
